docs-source/content/userguide/managing-domains/domain-events.md

```diff
@@ -29,14 +29,14 @@ The operator generates these event types, which indicate the following:
  *  `DomainProcessingFailed`: The operator has encountered a problem while it was processing the domain resource. The failure either could be a configuration error or a Kubernetes API error.
  *  `DomainProcessingRetrying`: The operator is going to retry the processing of a domain after it encountered an failure.
  *  `DomainProcessingCompleted`:  The operator successfully completed the processing of a domain resource.
- *  `DomainProcessingAborted`:  The operator stopped processing a domain when the operator encountered a fatal error or a failure that persisted after the specified maximum number of retries.
+ *  `DomainProcessingAborted`:  The operator stopped processing a domain when the operator encountered a fatal error, or a failure that persisted after the specified maximum number of retries.
```

Comment:
error, -> error (you don't need  comma here)

Reply:
IntelliJ showed a warning until I added the comma:). I'll remove it.